corehq/apps/export/models/new.py

```diff
@@ -251,9 +249,6 @@ class ExportInstance(Document):
     # Whether to split multiselects into multiple columns
     split_multiselects = BooleanProperty(default=False)
 
-    # Whether to automatically convert dates to excel dates
-    transform_dates = BooleanProperty(default=False)
```

Comment:
Yea it's an interesting one. In my mind the best option from a data model
perspective would be to have a 'global_transforms' property at the
ExportInstance level. Pros being that you don't have to update every column
to change the value and you don't get duplication in the data. The con is
that you have to then pass then around in the code.

Considering there is only 1 global transform option I don't think there is
can clear winner. If we had more global transform options I would go for
the approach described above.

Fine to leave this as is. One question though, how do you determine of the
check box needs to be selected when you load the screen for an existing
export? (Relying via email on my phone so don't have full diff handy)